google_takeout_parser/parse_json.py

```diff
@@ -259,6 +266,7 @@ def _parse_chrome_history(p: Path) -> Iterator[Res[ChromeHistory]]:
                 # and there's likely lots of items that aren't https
                 url=item["url"],
                 dt=time_naive.replace(tzinfo=timezone.utc),
+                pageTransition=item["page_transition"] if "page_transition" in item else None
```

Comment:
small nit -- best to use `item.get('page_transition')` , less code, works faster (especially matters since there could be lots of items in chrome history), and less copy pasting for key names :)

Reply:
gotcha I'll update this!